src/effects/backends/builtin/filtereffect.cpp

```diff
@@ -128,8 +128,8 @@ void FilterEffect::processChannel(
             double qmax = 4 - 2 / 0.6 * ratio;
             clampedQ = math_min(clampedQ, qmax);
         }
-        pState->m_pLowFilter->setFrequencyCorners(1, lpf, clampedQ);
-        pState->m_pHighFilter->setFrequencyCorners(1, hpf, clampedQ);
+        pState->m_pLowFilter->setFrequencyCorners(mixxx::audio::SampleRate(1), lpf, clampedQ);
```

Comment:
but the parameter is named samplerate. What should it be named instead if its not used as a samplerate?

Reply:
It can be refactored into using our mixxx::audio::SampleRate that is always in samples/s. 
Currently it looks it sets the filter to a 1/s sample rate.

Reply:
I'm unsure what you want me to do TBH.

Reply:
I am not sure if we have a common understanding about the issue here. Do we? 

I propose to revert the Fidlib related changes here and merge the rest of this PR quickly. This way we can consider the best solution for this issue in a smaller PR < 66 files .

Reply:
I checked in the fidlib code and the `sampleRate` parameter is actually passed as `double rate` parameter of the `fid_design()` method. In the error handling code, rate is used as the second argument of the error message `Frequency of %gHz out of range with sampling rate of %gHz`. So it seems indeed the `sampling rate`. No idea why we pass just `1` though.

Should I revert the changes nonetheless?

Reply:
So I think we have two options. 
1. Postpone the Fidlib related actions 
2. Use here the real 1/s base sample rate values via our semantic type and adjust the other parameters accordingly.

Reply:
Undoing all the fidlib-related changes turned out tedious, so I opted for 2. Please have a look at https://github.com/mixxxdj/mixxx/pull/11904/commits/99e1088b6c7566ec15ab23ff9153b197b75af3e6.